play-json/src/main/scala/com/lightbend/lagom/scaladsl/playjson/Serializers.scala

```diff
@@ -11,6 +11,30 @@ import scala.reflect.ClassTag
 object Serializers {
 
   /**
+   * Implicitly provided serializers
+   */
+  object Implicits {
+    implicit def optionFormat[A](implicit innerFormat: Format[A]): Format[Option[A]] =
```

Comment:
could this be `implicit class ... extends AnyVal`?
(not important since it's not in any hot path)